app/views/competitions/_nav.html.erb

```diff
@@ -267,11 +267,12 @@
         <% end %>
       <% end %>
 
-      <% @competition.warnings_for(current_user).each do |field, message| %>
-        <%= alert :warning, message, note: true %>
+      <% unless defined?(@skip_warnings )%>
```

Comment:
Why are you wrapping this in a `defined?` call? The variable `@skip_warnings` itself is a boolean, so `unless @skip_warnings` should do.

Reply:
I think that caused the tests to fail? `unless @skip_warnings` was my original try, let me try again

Reply:
Interestingly you can't do that because `@undefined_variable == false` -> false